ONSdigital/census-rm-toolbox
# Add basic audit logging to toolbox

# Motivation and Context
We need some audit logging for the toolbox, which is super powerful.

# What has changed
Added logging for logins, bash commands and psql.

# How to test?
Deploy a toolbox somewhere. Logon and give your name. Do some stuff (bash commands, dosql stuff) and then check `/home/toolbox/.audit/<name>` for the various things that you did. Because of the way bash works, the command history is only written after you exit the bash shell.

# Links
Trello: https://trello.com/c/Y95bUvnJ
